scripts/releaseChangelog.mts

```diff
@@ -239,12 +239,19 @@ function getComponentsFromLabels(labels: string[]): string[] {
     return [GENERAL_CHANGES_HEADER];
   }
 
+  const excludedScopeLabels = ['scope: support-infra', 'scope: code-infra', 'scope: docs-infra'];
```

Comment:
is this list axhaustive?

Reply:
Not even close. There are currently [100 "scope: " labels](https://github.com/mui/base-ui/labels?q=%22scope%3A+%22), but we don't use most of them, so it's good enough for now.

Reply:
I hope we can go back to using "component: " labels so this won't be necessary.